ChatQnA/docker_compose/amd/gpu/rocm-vllm/Dockerfile-vllm

```diff
@@ -0,0 +1,18 @@
+FROM rocm/vllm:rocm6.2_mi300_ubuntu20.04_py3.9_vllm_0.6.4
```

Comment:
let's put the dockerfile under `GenAIExamples/ChatQnA/`, it can be named as `Dockerfile.vllm_rocm`.

Reply:
Good. I will place the file according to the proposed path and adapt the scripts

Reply:
@chyundunovDatamonsters will you continue working on this PR?